tests/core/pyspec/eth2spec/test/helpers/voluntary_exits.py

```diff
@@ -23,3 +24,21 @@ def sign_voluntary_exit(spec, state, voluntary_exit, privkey):
         message=voluntary_exit,
         signature=bls.Sign(privkey, signing_root)
     )
+
+
+#
+# Helpers for applying effects of a voluntary exit
+#
+def get_exited_validators(spec, state):
+    current_epoch = spec.get_current_epoch(state)
+    return [index for (index, validator) in enumerate(state.validators) if validator.exit_epoch <= current_epoch]
+
+
+def exit_validators(spec, state, validator_count, rng=None):
```

Comment:
Is the following a poor pattern for some reason?
```suggestion
def exit_validators(spec, state, validator_count, rng=Random(1337)):
```

Reply:
i do it this way out of habit to avoid issues like this: https://stackoverflow.com/a/4841873

i think this will be fine though as the value is not mutable in a sense that would cause a problem like this

Reply:
fine either way. We use `rng=Random(num)` all over the place in the tests I beleive so just wanted  to make sure I wasn't doing something terrible